test/fork/migration-v2.js

```diff
@@ -590,6 +648,101 @@ describe('V2 upgrade', function () {
     expect(claimPayableAddressAfter).to.be.equal(AddressZero);
   });
 
+  it('compares storage of upgraded contracts', async function () {
+    // MCR
+    this.contractData.mcr.after.mcrFloorIncrementThreshold = await this.mcr.mcrFloorIncrementThreshold();
+    this.contractData.mcr.after.maxMCRFloorIncrement = await this.mcr.maxMCRFloorIncrement();
+    this.contractData.mcr.after.maxMCRIncrement = await this.mcr.maxMCRIncrement();
+    this.contractData.mcr.after.gearingFactor = await this.mcr.gearingFactor();
+    this.contractData.mcr.after.minUpdateTime = await this.mcr.mcrFloor();
+    this.contractData.mcr.after.mcr = await this.mcr.mcr();
+    this.contractData.mcr.after.desiredMCR = await this.mcr.desiredMCR();
+    this.contractData.mcr.after.lastUpdateTime = await this.mcr.lastUpdateTime();
+    this.contractData.mcr.after.previousMCR = await this.mcr.previousMCR();
+
+    Object.entries(this.contractData.mcr.before).forEach(([key, value]) => {
+      expect(this.contractData.mcr.after[key], `AssertionError: values of ${key} don't match\n`).to.be.equal(value);
+    });
+
+    // POOL
+    const assetsData = await Promise.all(
```

Comment:
can we make an assert here for the length or what assets we expect there to be?


The code is generalized so it would make it obvious everything that should there is in there